src/edu/wright/cs/raiderplanner/model/Room.java

```diff
@@ -46,46 +48,92 @@ protected void replace(VersionControlEntity receivedVce) {
 	// public methods
 
 	// getters
+	/**
+	 * Default method to return the building this room resides in.
+	 * @return building.
+	 */
 	public Building getBuilding() {
 		return building;
 	}
 
+	/**
+	 * Default method to return the room number.
+	 * @return roomNumber.
+	 */
 	public String getRoomNumber() {
 		return roomNumber;
 	}
 
+	/**
+	 * Method to output the name and room number of a person.
+	 * If the room does not have a building number, the
+	 * output message displays "name - Unknown Building".
+	 * If the building is not null, the output message displays
+	 * "name ( room ) located in building" where room represents
+	 * the room number and building represents the building object.
+	 * @return String.
+	 */
 	public String getLocation() {
-		return name + "( " + roomNumber + " )";
+		if (building == null) {
+			return name + " - Unknown Building";
+		} else {
+			return name + "( " + roomNumber + " ) located in " + building.toString();
+		}
 	}
 
+	/**
+	 * Overrides the toString method for the Room object.
+	 * If the building is null, the output is "name - Unknown Building",
+	 * otherwise the output is formatted as "name ( room ) located in building"
+	 * where room represents the room number and building is the name of the
+	 * building object.
+	 * @return String
+	 */
 	@Override
 	public String toString() {
-		// TODO: Figure out if there is a better approach here, perhaps including
-		// "unknown" or similar if build is null; also make output consistent with
-		// other methods, like getLocation() and getRoomNumber()
 		if (building == null) {
-			return name + "( " + roomNumber + " )";
+			return name + " - Unknown Building";
 		} else {
 			return name + "( " + roomNumber + " ) located in " + building.toString();
 		}
 	}
 
 	// setters
+	/**
+	 * Sets a value for a building.
+	 * @param newBuilding the new building being assigned a value.
```

Comment:
The `@param` text does not make much sense. Something like `set this room's building to the specified value` would be better.